packages/commonwealth/client/scripts/views/modals/AuthModal/useAuthentication.tsx

```diff
@@ -402,7 +403,7 @@ const useAuthentication = (props: UseAuthenticationProps) => {
       // @ts-expect-error StrictNullChecks
       await verifySession(session);
       // @ts-expect-error <StrictNullChecks>
-      await onLogInWithAccount(account, false, true, false);
```

Comment:
not sure why we removed the last boolean?

Reply:
last argument is `shouldRedrawApp`. It was removed whatsoever